content/guides/react/hooks.mdx

```diff
@@ -0,0 +1,12 @@
+---
+title: Primer React hooks
```

Comment:
I think this page would be greatly improved if we treated more like the other listing pages (for example, [Components](https://primer-b77ed95906-26441320.drafts.github.io/components)).

We could render each hook name as a heading, and then provide a brief description.